database/migration/migrateData.ts

```diff
@@ -619,6 +619,12 @@ const migrateData = async () => {
     `)
 
     console.log(' - Simplify how we handle "locked" reviews')
+    // We need to drop these first so we can subsequently manipulate
+    // review_status and review_status_old.
+    await DB.changeSchema(`
+      DROP VIEW IF EXISTS assigned_sections_by_stage_and_level CASCADE;
+      DROP TYPE IF EXISTS public.review_status_old;
```

Comment:
We don't strictly need to drop the type here (as it should never exist), but some of the recent snapshots have it still there due to the aforementioned failures, so safer to make sure we drop it too.